opencog/atoms/bind/PatternLink.cc

```diff
@@ -45,13 +45,6 @@ void PatternLink::common_init(void)
 	                 _fixed, _virtual, _pat.black);
 	_num_virts = _virtual.size();
 
-	// unbundle_virtual does not handle connectives. Here, we assume that
-	// we are being run with the DefaultPatternMatchCB, and so we assume
-	// that the logical connectives are AndLink, OrLink and NotLink.
-	// Tweak the evaluatable_holders to reflect this.
-	std::set<Type> connectives({AND_LINK, OR_LINK, NOT_LINK});
-	trace_connectives(connectives, _pat.clauses);
```

Comment:
Removing this looks wrong.  I made the connectives configurable, because different logic would use different connectives. For example, for probability theory, the connectives would be SET_UNION_LINK, SET_INTERSECTION_LINK, and SET_COMPLEMENT_LINK being the analogs of boolean OR, AND, NOT.  Other logics would use different connectives. 

Because PLN has a "special" interlretation of AndLink, OrLink, NotLink, then a better solution that avoids conflich with PLN would be to change the above to BOOLEAN_AND_LINK, BOOLEAN_OR_LINK, BOOLEAN_NOT_LINK, to make it clear that crsip-logic is being used to combine the results of evaluating terms.